apps/passport-client/src/localstorage.ts

```diff
@@ -27,6 +29,21 @@ export async function loadPCDs(): Promise<PCDCollection> {
   );
 }
 
+export async function downloadAndDecryptStorage(
```

Comment:
i dont think this function should be in this file

Reply:
moved to `endToEndEncryptionApi.ts`